qriollo.rb

```diff
@@ -0,0 +1,30 @@
+# coding: utf-8
+class Qriollo < Formula
+  desc "An impure functional language, based on Rioplatense Spanish."
+  homepage "https://qriollo.github.io"
+  url "https://qriollo.github.io/Qriollo-0.91.tar.gz"
+  sha256 "c8357af8254a082d8e4da1de1bbf13bee27cfde8adb31ea0a5a0966bfbb7b28d"
+  head "https://github.com/qriollo/qriollo.git"
+
+  depends_on "ghc"
```

Comment:
Oh, wow, that particular oversight is something *I* should have caught!  I'll fix it; thanks!